src/ResourceManager/Network/Commands.Network/help/New-AzureRmPublicIpTag.md

```diff
@@ -0,0 +1,126 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: 
```

Comment:
Please add the online version (similar format to the other files)

Reply:
done